app.rb

```diff
@@ -29,6 +30,11 @@ class BnB < Sinatra::Base
     erb(:users_new)
   end
 
+  post '/users/new' do
+    User.create(params['name'], params['email'], params['password'])
```

Comment:
should the params be the same is and the of the fields in the form, otherwise the params will retrieve nil data?